### PR TITLE
#565 | how to add export env var

### PR DESCRIPTION
**Pull request by hoavt**
connect #565

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -374,15 +374,23 @@ https://help.github.com/articles/connecting-to-github-with-ssh/
 
    .. note::
 
-      On Windows, you must always use ``Cygwin Terminal``, not ``Git Bash``.
-      If you have SSH configured and vagrant is still not working on Windows, set the
-      `VAGRANT_PREFER_SYSTEM_BIN` environment variable to true:
+      - On Windows, you must always use ``Cygwin Terminal``, not ``Git Bash
```
Comment (hoatle) [inline docs/getting_started.rst]: not \``Git Bash``.

File: docs/getting_started.rst [hunk 2]
```diff
@@ -374,15 +374,23 @@ https://help.github.com/articles/connecting-to-github-with-ssh/
...
+
+      - If you have SSH configured and `vagrant` is still not working on Windows, you should add the
+        `export VAGRANT_PREFER_SYSTEM_BIN=true` environment variable to the `./bash_profile` file, that helps you
```
Comment (hoatle) [inline docs/getting_started.rst]: \``export VAGRANT_PREFER_SYSTEM_BIN=true``

\``.bash_profile``

File: docs/getting_started.rst [hunk 3]
```diff
@@ -374,15 +374,23 @@ https://help.github.com/articles/connecting-to-github-with-ssh/
...
-          $ vagrant status
+          $ cat >> ~/.bash_profile
+
+      Type `export VAGRANT_PREFER_SYSTEM_BIN=true` and press `Ctrl + D`, then run:
+
+       ..  code-block:: bash
+
+           $ source ~/.bash_profile
 
+      Now, open the terminal and run the `export` command to check if the variable is added successfully
```
Comment (hoatle) [inline docs/getting_started.rst]: \``export``

**Review by hoatle — CHANGES_REQUESTED** [review]
@hoavt pls check my comments, you always should open the .rst file on github to see the output result to prevent these kinds of mentioned problems.

**Comment by hoavt** [comment]
Applied your comments. Please check @hoatle.

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**